README.md

```diff
@@ -172,6 +172,52 @@ app.use(g2o({
 }));
 ```
 
+### unauthenticatedStatusCode
+
+Optional; `Integer` (default: 401)
+
+If provided, the status code will be used as the response status code for when the authentication fails.
+
+### onUnauthenticated
+
+Optional; `Function` (default: noop)
+
+If provided, the function will be called after all checks have been completed and found to be unauthenticated.
```

Comment:
I like this idea.  I did not like how the two were related but not.  This is an easier to understand solution.